source/components/ansible-setup.rst

```diff
@@ -59,6 +59,19 @@ Allan have access to it. Stash the password in file named
 You will also need sudo access on the hosts you want to change if that change
 requires privilege escalation.
 
+.. note::
+    The ``ansible-conda`` submodule should be copied locally using the
+    following command:
+
+    .. code-block:: bash
+        git submodule update --init --recursive
+.. note::
```

Comment:
Same here. FWIW I'm not sure these need to be offset as notes -- maybe just more prose? -- but I leave that up to you.

Reply:
I prefer to leave them as notes, but not strongly if others are adamant they need to be changed let me know and I will change them